### ISSUE TITLE
No matter how I move, the step count always shows as 0. 

### ISSUE THREAD

**Issue by Epoch2022-iOS** [SecUSo/privacy-friendly-pedometer#154]
Even when I turn on the "Permanent step counter" switch, the step count does not increase!

see the screenshots:
![WechatIMG445](https://github.com/SecUSo/privacy-friendly-pedometer/assets/24999102/1e9a5f6f-9a17-4094-817f-cbffdda8b7a7)
![WechatIMG446](https://github.com/SecUSo/privacy-friendly-pedometer/assets/24999102/0013b23c-6416-4048-af53-cd5c971837b5)

**Comment by Epoch2022-iOS** [comment]
make a breakpoint,I find this reason:
![WechatIMG36846](https://github.com/SecUSo/privacy-friendly-pedometer/assets/24999102/f4cc0e51-5bb9-468c-8898-33dafc8d04b7)

**Comment by morckx** [comment]
Have you tried using your phone's step counting hardware as counter?

**Comment by Epoch2022-iOS** [comment]
> Have you tried using your phone's step counting hardware as counter?
Yes, I tried using the counting hardware on my phone, but it prompted that my phone doesn't support counting hardware. So, I just used the Permanent Step Counter to track my steps.

**Comment by rdiez** [comment]
I just tried version 3.1.0 from F-Droid on my Galaxy XCover 4s, and it does not count any steps. I tried setting the sensitivity to the highest to no avail.

F-Droid has another pedometer app from Levente Bagi, but it was automatically blocked because of alleged security vulnerabilities. No wonder, that app hasn't been updated since 2011.

There is another pedometer app on F-Droid from www.j4velin.de , but F-Droid says it is not compatibly with my phone. It also hasn't been updated for 5 years.

So I cannot find any working open source app for this purpose. 8-(
